lib/logstash/codecs/cef.rb

```diff
@@ -266,13 +269,16 @@ def get_value(fieldname, event)
 
     return nil if val.nil?
 
+    key = "#{sanitize_extension_key(fieldname)}"
```

Comment:
since `sanitize_extension_key(fieldname)` returns a `String` there is no need to use string interpolation. You could just write:
```rb
key = sanitize_extension_key(fieldname)
```

Reply:
Ok. Will adjust my pull request.